build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
     id 'jacoco'
 }
 
-mainClassName = 'seedu.address.Main'
+mainClassName = 'seedu.intern.Main'
```

Comment:
I think these may change in the future, `intern` is not confirmed yet, so maybe no need to change now? I saw this in a few other places too!

Reply:
think we can leave it as is first, when the time comes I can change it to whatever we want